styles.css

```diff
@@ -92,11 +119,9 @@ a[href="index.html"]{
 
 .social a{
 	height:30px;
-	background-repeat: no-repeat;
-	background-position: center;
+	background-repeat:no-repeat;
```

Comment:
what background image do you want to `no-repeat`?

Reply:
This line is causing the image to display once only